From 0893740293925814e55b5186bee4dfa41501816a Mon Sep 17 00:00:00 2001
From: Nathan Rossi <nathan@nathanrossi.com>
Date: Thu, 17 Mar 2016 17:43:58 +1000
Subject: [PATCH] HACK: ARM: zynq: Force MMC boot regardless of boot config

Parallella boots SPL from QSPI but the u-boot and system images are
located on SD.

Signed-off-by: Nathan Rossi <nathan@nathanrossi.com>
Upstream-Status: Inappropriate [Hack to force for specific board]
---
 arch/arm/mach-zynq/spl.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/mach-zynq/spl.c b/arch/arm/mach-zynq/spl.c
index d09141c3bc..d6faf24c77 100644
--- a/arch/arm/mach-zynq/spl.c
+++ b/arch/arm/mach-zynq/spl.c
@@ -69,6 +69,9 @@ u32 spl_boot_device(void)
 		hang();
 	}
 
+	puts("force mmc boot\n");
+	mode = BOOT_DEVICE_MMC1;
+
 	return mode;
 }
 
